components/brave_rewards/resources/ui/components/settingsPage.tsx

```diff
@@ -53,6 +54,7 @@ class SettingsPage extends React.Component<Props, {}> {
     this.actions.getWalletProperties()
     this.balanceTimerId = setInterval(() => {
       this.actions.getWalletProperties()
+      this.getAdsNotifications()
```

Comment:
I think we shouldn't be pulling it every min. We should detect when this data changes (ad is viewed/clicked and that should trigger update).

Reply:
@NejcZdovc It makes more sense to update on data changes. @NejcZdovc @jsecretan with timelines is this something that should be changed now, or can a ticket be raised?

Reply:
I think we should change now as for someone that don't have ads enabled it will pull all the time

Reply:
👍